src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/FormViewNetworkWideFields.tsx

```diff
@@ -176,17 +164,51 @@ export const ProtocolTypeSelect = () => {
       setProtocolType(newProtocolType);
     }
   };
-
+  const isIpv4Selected = protocolType === 'ipv4';
   return (
     <>
-      <OcmSelectField
+      <FormGroup
+        fieldId={getFieldId(selectFieldName, 'radio')}
+        isInline
         label="Internet protocol version"
-        options={protocolVersionOptions}
-        name={selectFieldName}
-        callFormikOnChange={false}
-        onChange={onChange}
-        data-testid="select-protocol-version"
-      />
+        isRequired
+      >
+        <OcmRadioField
+          label={
+            <>
+              {getProtocolVersionLabel(ProtocolVersion.ipv4)}{' '}
+              <PopoverIcon
+                noVerticalAlign
+                bodyContent="Select this when your hosts are using only IPv4."
+              />
+            </>
+          }
+          name={selectFieldName}
+          data-testid="select-ipv4"
+          id="select-ipv4"
+          value="ipv4"
+          isChecked={isIpv4Selected}
+          onChange={onChange}
```

Comment:
You can probably move the `onChange` to the `FormGroup` and keep the previous signature , as you are not using `checked` for anything. (See `StackTypeControl`)